internal/pkg/agent/application/coordinator/coordinator.go

```diff
@@ -1740,7 +1740,7 @@ func (c *Coordinator) updateManagersWithConfig(model *component.Model) {
 		for _, comp := range otelModel.Components {
 			componentIDs = append(componentIDs, comp.ID)
 		}
-		c.logger.With("component_ids", componentIDs).Warn("The Otel runtime manager is HIGHLY EXPERIMENTAL and only intended for testing. Use at your own risk.")
+		c.logger.With("component_ids", componentIDs).Warn("Using OpenTelemetry collector runtime.")
```

Comment:
maybe it's not a warning but an info at this time ?

```suggestion
		c.logger.With("component_ids", componentIDs).Info("Using OpenTelemetry collector runtime.")
```

Reply:
I agree, not a warning anymore. Thanks.